src/rdf4cpp/rdf/datatypes/registry/LiteralDatatypeImpl.hpp

```diff
@@ -312,19 +312,14 @@ struct LiteralDatatypeImpl : capabilities::Default<type_iri>, Capabilities<type_
                   "Mismatch between declared and actual fixed id mapping state. "
                   "Hint: maybe you forgot declare the fixed id or to add the FixedId capability. "
                   "Note: this would cause inconsistency between registry and node storage");
-protected:
-    static std::nullptr_t init() noexcept;
-    inline static std::nullptr_t const dummy = init();
+private:
```

Comment:
What's the rational behind this change? How is it different from what was there before?

Reply:
Well, it simly doesn't work without that change. Tbh I have no idea why it doesn't. But it seems the object is not enough for the linker to no drop the symbol for the static member. With a non-static member it doesn't drop
it for some reason.